row_cli.py

```diff
@@ -135,8 +126,35 @@ def main():
 
         return circles
 
-    if args["detect"] and args["characters"]:
-        row.create_batch_character_detection_job(args["--from"])
+    if args["process"] and args["images"]:
+        return row.mosaic_all_circles(
+            args["--job"],
+            args["--from"],
+            args["--save-to"],
+            args["--index"],
+            int(args["--task-index"]),
+            int(args["--instances"]),
+            int(args["--file-count"]),
+        )
+
+    if args["process"] and args["circles"]:
+        inputs = SimpleNamespace(
+            job_name=args["--job"],
+            input_bucket=args["--from"],
+            output_location=args["--save-to"],
+            file_index=args["--index"],
+            task_index=int(args["--task-index"]),
+            task_count=int(args["--instances"]),
+            total_size=int(args["--file-count"]),
+            project_number=int(args["--project"]),
+            processor_id=int(args["--processor"]),
+        )
+
+        results = row.ocr_all_mosaics(inputs)
+
+        print(f"operation finished with status {results}")
```

Comment:
question: `results` is the dataframe passed out of `ocr_all_mosaics`, right? What will that look like when printed here?  Is it just for the CLI and not the logging?

Reply:
it's actually a list of lists...

Reply:
Oh, that's right.  So it just prints the list (of lists) to the console when running the CLI?